runtests.py

```diff
@@ -183,7 +183,7 @@ def main(argv):
     if args.shell:
         shell = os.environ.get('SHELL', 'sh')
         print("Spawning a Unix shell...")
-        os.execv(shell, [shell] + extra_argv)
+        subprocess.check_call([shell] + extra_argv)
         sys.exit(1)
```

Comment:
Is this still correct? The documentation on os.execv says

> These functions all execute a new program, replacing the current process; they do not return. 

So I'm guessing that the `sys.exit(1)` is only there in case the `execv` call fails.